### PR TITLE
#167313402  user profile CRUD operation

### PR DESCRIPTION
**Pull request by salviosage**
#### What does this PR do?

- create  user profile 

#### Description of Task to be completed?

- user should be able to create a profile

- user should be able to update their profile

- user should be able to view other peoples profiles 

#### How should this be manually tested?

- by cloning this repo and switch to this branch 

- in your dotenv file, you should have cloudinary configuration including  `CLOUDINARY_NAME`,
  `CLOUDINARY_API_KEY`,
 `CLOUDINARY_SECRET_KEY`,

- run migration and seed then dev scripts 

- after you can login and use token to create a profile by making  a PUT request to  `/profile`  route

- you can view your profile by making  a GET request to `/profile/ `route 

- you can view someone profile by making  a GET request to `/profile/: username`   route 

#### What are the relevant pivotal tracker stories?
[#167313402](https://www.pivotaltracker.com/story/show/167313402)

#### Screenshots (if appropriate)
image
image
image
image
image

### CODE REVIEW

**Review by minega25 — CHANGES_REQUESTED** [review]
@salviosage Would you kindly remove the binary files like DS_Store and others because I think they are not supposed to be merged to develop

**Comment by salviosage** [comment]
> DS_Store

done.

**Review by habinezadalvan — APPROVED** [review]

**Review by minega25 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -38,11 +39,13 @@
     "express-async-errors": "^3.1.1",
     "express-jwt": "^5.3.1",
     "express-session": "^1.15.6",
+    "fs": "0.0.1-security",
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.5.1",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
     "multer": "^1.4.2",
+    "multer-storage-cloudinary": "^2.2.1",
```
Comment (Mnickii) [inline package.json]: Where is this used? Can't see where it's required in the code.

**Review by Mnickii — APPROVED** [review]
LGTM, but a comment below

File: package.json (hunk 1 above)
Comment (salviosage) [reply package.json]: > Where is this used? Can't see where it's required in the code.

oh I used it before  forgot to remove it, I removed it

**Pull request merged**